MSProject/DAO/WBBookDAO.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  WBBookDAO.swift
+//  MSProject
+//
+//  Created by Matias Spinelli on 07/06/2019.
+//  Copyright © 2019 Wolox. All rights reserved.
+//
+
+import UIKit
+
+enum SortMethod {
+    case id
+    case title
+    case author
+    case genre
+    case year
+}
+
+class WBBookDAO: NSObject {
```

Comment:
I think that this whole class can be implemented in either an extension or inside a ViewModel. Maybe something that can be called like `_viewModel.getBooksSorted(by: .author)` would be really nice. But I don't know if you can do that if you really need something that sorts the books in every single view.

Reply:
at the time, I had thought this class with much more features. now it is kind a helper.